client/src/store/modules/eventTypes/actions.js

```diff
@@ -7,14 +7,22 @@ import * as loaderMutations from '@/store/modules/loader/types/mutations';
 export default {
     [actions.FETCH_EVENT_TYPES]: async (
         { commit, dispatch },
-        searchString = ''
+        data = { searchString: '' }
     ) => {
         commit('loader/' + loaderMutations.SET_LOADING, true, { root: true });
         try {
             const eventTypes = await eventTypesService.fetchAllEventTypes(
-                searchString
+                data.searchString,
+                data.page
             );
-            commit(mutations.SET_EVENT_TYPES, eventTypes);
+            if (data.searchString) {
+                if (data.page === 1) {
+                    commit(mutations.CLEAR_EVENT_TYPES);
+                }
+                commit(mutations.SET_EVENT_TYPES, eventTypes);
+            } else {
+                commit(mutations.SET_EVENT_TYPES, eventTypes);
+            }
```

Comment:
There is no need for an if-else wrapper because this action `commit(mutations.SET_EVENT_TYPES, eventTypes);` will be executed for any outcome.   
This code is enough
`if (data.page === 1) {
                    commit(mutations.CLEAR_EVENT_TYPES);
                }
                commit(mutations.SET_EVENT_TYPES, eventTypes);`